server/service/src/processors/transfer/shipment/create_inbound_shipment.rs

```diff
@@ -78,6 +78,26 @@ impl ShipmentTransferProcessor for CreateInboundShipmentProcessor {
 
         InvoiceRowRepository::new(connection).upsert_one(&new_inbound_shipment)?;
 
+        if outbound_shipment.invoice_row.status.clone() == InvoiceRowStatus::Picked {
+            system_log_entry(
+                connection,
+                LogType::InvoiceCreated,
+                Some(outbound_shipment.invoice_row.store_id.clone()),
+                Some(new_inbound_shipment.id.clone()),
+            )?;
+        }
+
+        system_log_entry(
+            connection,
+            match outbound_shipment.invoice_row.status.clone() {
+                InvoiceRowStatus::Picked => LogType::InvoiceStatusPicked,
```

Comment:
I wonder if we can have just one LogType::from_invoice_status method that we use for all invoice update logs. 
I see that in service update method for shipments we match on input.status*, I think the logic there should be, did status change ? if yes lets log the new status (from invoice.status). I think it's worth updating this in this PR.

*input.status (does this mean if a comment for example is updated, we log that status was also changed ?),

Reply:
[2fb3de6](https://github.com/openmsupply/open-msupply/pull/666/commits/2fb3de6def14108b24c3a8d6045947d65b4d4401)